pkg/scheduler/eventhandlers.go

```diff
@@ -163,6 +164,10 @@ func (sched *Scheduler) updatePodInSchedulingQueue(oldObj, newObj interface{}) {
 		return
 	}
 
+	for _, evt := range framework.PodSchedulingPropertiesChange(newPod, oldPod) {
+		defer metrics.EventHandlingLatency.WithLabelValues(evt.Label).Observe(metrics.SinceInSeconds(start))
+	}
```

Comment:
We have two handlers for pod update: `updatePodInSchedulingQueue` for unscheduled pods and `updateNodeInCache` for scheduled. Shouldn't we emphasize in metric labels which latency we measure?

Reply:
Yeah, good point. Probably we should create different labels; ones for assigned pods and the others for unsched Pods?

Reply:
I'll sort out the label stuff in a different PR, probably along with the event list cleanup too which we talked somewhere before.

Reply:
With https://github.com/kubernetes/kubernetes/pull/127797, the metrics will get different labels, either AssignedPodXXX or UnschedulablePodXXX.